.travis.yml

```diff
@@ -20,7 +20,12 @@ install:
 # some tests such as zram.bats may fail if we run in emulation mode (testing for ARM emulated on x86 Travis cloud servers)
   - sudo apt-get install virt-what && VIRT=$(sudo virt-what) && echo "Travis VM running on ${VIRT:-native HW}."
   - docker run --rm --privileged multiarch/qemu-user-static:register --reset
-  - docker build -t openhabian/rpi-openhabian -f Dockerfile .
+# prepare configuration for tests, select debug level here:
+  - #sed -iE 's#mode=.*$#mode=unattended_debug#' openhabian.conf.dist
+  - sed -iE 's#mode=.*$#mode=debug_maximum#' openhabian.conf.dist
+  - sed -i 's#repositoryurl=https://github.com/openhab/openhabian.git#repositoryurl=https://github.com/'${TRAVIS_REPO_SLUG}'.git#' build-image/openhabian.conf
+  - sed -i 's#clonebranch=master#clonebranch='${TRAVIS_BRANCH}'#' build-image/openhabian.conf
+  - docker build -t openhabian/rpi-openhabian -f Dockerfile.openhabian .
```

Comment:
getting confused, which file do you want to modify? (see Dockerfile below, you are copying both?!)
Before I had the impression you wanted to base the Docker test on conf.dist.

Reply:
initially yes. Now changed, using build-image/openhabian.conf